systers_portal/meetup/tests/test_views.py

```diff
@@ -1831,3 +1833,53 @@ def check_message_in_response(self, _message_tag=None, _message=None, status_cod
         for message in response.context['messages']:
             self.assertEqual(message.tags, _message_tag)
             self.assertTrue(_message in message.message)
+
+
+class ApiForVmsViewTestCase(APITestCase, TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username='foo', password='foobar',
+                                             email='user@test.com')
+        self.systers_user = SystersUser.objects.get(user=self.user)
+        country = Country.objects.create(name='Bar', continent='AS')
```

Comment:
I don't know much about portal but should this be `self.country`?

Reply:
No, as the field isn't directly used as a field , self doesn't need to be attached .  Also, you can see while testing the meetup location which is the first test in the series, the same format has been used.